src/structures/InteractionCollector.js

```diff
@@ -19,8 +19,8 @@ const { InteractionTypes, MessageComponentTypes } = require('../util/Constants')
 
 /**
  * Collects interactions.
- * Will automatically stop if the message (`'messageDelete'`),
- * channel (`'channelDelete'`), or guild (`'guildDelete'`) are deleted.
+ * Will automatically stop if the message ({@link Client#messageDelete messageDelete}),
+ * channel ({@link Client#channelDelete channelDelete}), or guild ({@link Client#guildDelete guildDelete}) are deleted.
```

Comment:
```suggestion
 * channel ({@link Client#channelDelete channelDelete}), or guild ({@link Client#guildDelete guildDelete}) is deleted.
```

Reply:
Nope, that is right (if the message, channel or guild are deleted)

Reply:
This is a list of multiple items, `are` is correct

Reply:
Idk man if you can find me an article on that sure but are sounds better to me

Reply:
Here are a couple:

- https://owl.purdue.edu/owl/general_writing/grammar/subject_verb_agreement.html
- https://www.grammarbook.com/grammar/subjectverbagree.asp

Reply:
Grammarly also agrees that it should be `is`. You are only talking about one thing after the `or`, so it should be `is`, not `are`.

Examples: 
If the channel, or guild are deleted ❌
If the message and channel are deleted ✔️
If the message, or guild and channel are deleted ✔️

The second and third examples make sense because you are referring to both; however, you aren't with the first example.